optimum/pipelines.py

```diff
@@ -96,6 +97,12 @@
             "default": "t5-small",
             "type": "text",
         },
+        "fill-mask": {
```

Comment:
remove

Reply:
fixed in [4013264](https://github.com/huggingface/optimum/pull/423/commits/4013264247559c19d3495a9f4636b509a5243ba9)